.github/workflows/tf_apply.yml

```diff
@@ -46,3 +46,7 @@ jobs:
       - name: Apply hosted_zone
         working-directory: terragrunt/env/staging/hosted_zone
         run: terragrunt apply --terragrunt-non-interactive -auto-approve
+      
+      - name: Apply netwrok 
```

Comment:
Typo, non-blocking

Reply:
Thanks Calvin! Totally did not noticed this. It has been corrected.